### ISSUE TITLE
Bunne2023 implementation

### ISSUE THREAD

**Issue by yCebo** [yCebo/OT_analysis#1]

**Comment by yCebo** [comment]
- The config structure
  - data:
    - features: text
    - path: h5ad
    - condition: drug/treatment
    - source: control
    - type: cell

  - dataloader:
    - batch_size
    - shuffle
  
  - datasplit:
    - groupby: condition
    - name
    - test_size

**Comment by yCebo** [comment]
- Model:
    - name: cell_ot
    - hidden_units: [64, 64, 64, 64]
    - latent_dim: 50
    - softplus_W_kernels: false

    - g:
        - fnorm_penalty: 1
 
    - kernel_init_fxn:
        - b: 0.1
        - name: uniform

- optim:
    - optimizer: Adam
    - lr: 0.0001
    - beta1: 0.5
    - beta2: 0.9
    - weight_decay: 0

- training:
    - n_iters: 100000
    - n_inner_iters: 10
    - cache_freq: 1000
    - eval_freq: 250
    - logs_freq: 50
